docs/governance/README.md

```diff
@@ -0,0 +1,129 @@
+environment (for instance, Xamarin for iOS cannot perform runtime code
+generation, based on Apple's app store policy).
+
+To ensure we don't end up adding large chunks of API surface that cannot be
+implemented, we have a review board that has to sign-off on API additions to the
+.NET Standard. The board comprises representatives from the following groups:
+
+  * **.NET platform**. The rationale here is that most, if not all, of the APIs
+    that are part of .NET Standard are implemented and evolved by the .NET
+    platform team.
+  * **Xamarin & Mono**. While Mono mostly copies code from the .NET Framework &
+    .NET Core base class libraries, changes and extensions can impact their
+    ability to support the .NET Standard. Thus, we need to coordinate any
+    changes with Mono.
+  * **Unity**. Same rationale as for Xamarin & Mono.
+
+The chairman of the review board is [@migueldeicaza](https://github.com/migueldeicaza).
+For the most part, we strive to make decisions based on consensus, but as it is
+with every group of engineers, consensus can be impossible to achieve at times
+so we need to have an ultimate tie breaker. And Miguel has a lot of expertise
+and experience building .NET implementations that are supported by multiple
+parties.
+
+Please note that the member list isn't meant to be closed: as more platform
+vendors and API drivers appear, the review board will expand accordingly.
+
+## Process
+
+* **Anybody can submit proposals** for API additions to the .NET Standard.
+  Those will be tracked as issues and labeled with the [netstandard-api] label.
+* **We'll track features, no individual APIs**. In order to keep bookkeeping we
+  generally will usually file an issue per feature, rather than per API. The
+  issue description will then contain the APIs that should be added.
+* **New members on standardized types are automatically considered**. To prevent
+  accidental fragmentation, we'll automatically consider all members added by
+  any .NET implementation on types that are already in the standard. The
+  rationale here is that divergence at that the member level is not desirable
+  and unless there is something wrong with the API it's likely a good addition.
+* **Acceptance requires**
+  - A **sponsorship from a board member**. That person will be assigned the
+    issue and is expected to shepherd the issue until it's either accepted or
+    rejected. If no board member is willing to sponsor the proposal, it's
+    considered rejected.
+  - A **stable implementation** in at least one .NET implementation. The
+    implementation must be licensed under an open source license that is
+    compatible with MIT. This will allow other .NET implementations to jump-
+    start their own implementations or simply take the feature as-is.
+* **.NET Standard updates are planned** and will generally follow a set of
+  themes. We avoid grab-bag releases of .NET Standard and instead try to define
```

Comment:
What are "grab-bag releases"?

Reply:
Grab-bag releases where you a large number of small-ish things that don't follow a set of scenarios. That makes it hard to prioritize and make trade-off decisions.

Reply:
👍 thanks. I'm not sure the term "grab-bag" is recognised outside North America.

Reply:
How about "potpourri"-style releases?

Reply:
Reworded to:

> **.NET Standard updates are planned** and will generally follow a set of themes. We avoid releases with a large number of tiny features that aren't part of a common set of scenarions (we call them "grab-bag-style releases"). Instead, we try to define a set of goals that describe what kind of feature areas a particular .NET Standard version provides. This simplifies answering the question which .NET Standard a given library should depend on. It also makes it easier for .NET implementations to decide whether it's worth implementing a higher version of .NET Standard.